src/components/Header/Header.module.css

```
@@ -1,20 +1,126 @@
 .header {
-    background-color: #f8f9fa;
-    padding: 10px 20px;
-    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
-    position: fixed;
-    width: 100%;
-    top: 0;
-    left: 0;
-    z-index: 1000;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 48px 24px;
+  background-color: var(--midnight-green);
+  color: var(--primary-white);
+}
+
+.hamburgerIcon {
+  width: 20px;
+  height: 17px;
+  cursor: pointer;
+}
+
+.sideDrawer {
+  position: fixed;
+  right: -100%;
+  width: 255px;
+  height: 100%;
+  background-color: var(--police-blue);
+  transition: left 0.3s ease-in-out;
+  z-index: 200;
+  color: var(--primary-white);
+  background-image: url("../../assets/bg-pattern-about-1-mobile-nav-1.svg");
+  background-position-x: calc(100% + 100px);
+  background-position-y: bottom;
+  background-repeat: no-repeat;
+}
+
+.open {
+  right: 0;
+}
+
+.nav {
+  display: none;
+}
+
+.mobileNav {
+  padding-block: 56px;
+  padding-right: 25px;
+  padding-left: 48px;
+}
+
+.mobileNav ul,
+.nav ul {
+  list-style: none;
+}
+
+.mobileNav li,
+.nav li {
+  margin: 24px 0;
+}
+
+.mobileNav a,
+.nav a {
+  color: var(--primary-white);
+  text-decoration: none;
+}
+
+.mobileNav a:hover,
+.nav a:hover {
+  color: var(--coral-color);
+}
+
+.backdrop {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.5);
+  z-index: 100;
+}
+
+.contact {
+  margin-top: 21px;
+  width: 159px;
+}
+
+.closeIcon {
+  cursor: pointer;
+  display: block;
+  margin-left: auto;
+}
+
+@media (min-width: 768px) {
+  .header {
+    padding-inline: 39px;
+    align-items: center;
+    justify-content: space-between;
   }
-  
-  .header nav ul {
-    list-style: none;
+
+  .logo {
+    width: 128px;
+    height: 32px;
+    margin-right: 28px;
+  }
+
+  .hamburgerIcon {
+    display: none;
+  }
+
+  .navLink {
+    color: var(--primary-white);
+    text-decoration: none;
+    margin-inline: 20px;
+  }
+
+  .nav {
+    display: block;
+  }
+
+  .navLinks {
     display: flex;
-    justify-content: space-around;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+  }
+}
+
+@media (min-width: 960px) {
+  .header {
+    padding-inline: 7rem;
```

For desktop the horizontal paddings need to be equal for the header and the first section contents on all 3 pages right now it is different for all. The current inline value for header is slightly smaller
```suggestion
    padding-inline: 165px;
```

---

This is actually an issue for aligning the inline padding of _all_ sections, not just top sections. There is not a coherent take on how the gutter space (empty space on the left/right edges) is handled across components. This might be best handled in another separate ticket.